README.md

```diff
@@ -152,7 +152,8 @@ There are 8 schemas on the connected chain.
 
 ## Use with Docker
 
-This repo includes a docker image to push a [Frequency instant-seal-node](https://hub.docker.com/r/frequencychain/instant-seal-node) with the schemas deployed on top of it to docker hub under `dsnp/instant-seal-node-with-deployed-schemas`.
+This repo deploys `dsnp/instant-seal-node-with-deployed-schemas` to Docker Hub.
+It is based on a [Frequency Standalone Docker](https://hub.docker.com/r/frequencychain/standalone-node) with the schemas automatically deployed on top of it.
```

Comment:
It's fairly obvious from the name, but do we want to explicitly say it runs with the same default argument values as `standalone-node` (including instant sealing)?

Reply:
Updated.